shared/src/packages/trucks/trucks.ts

```diff
@@ -0,0 +1,18 @@
+export {
+  FormLabel,
+  FormName,
+  TruckApiPath,
+  TruckCapacity,
+  TruckManufacturer,
+  TruckPricePerKm,
+  TruckTowType,
+  TruckValidationMessage,
+  TruckYear,
+} from './libs/enums/enums.js';
+export { type TruckEntity } from './libs/types/types.js';
+export {
+  LICENSE_PLATE_NUMBER_REGEX,
```

Comment:
this is exported from validation-schemas/validation-schemas - and it's weird

Reply:
I thought that it would be clearer to import from the index file, and everything should be separated into packages, shouldn't it?